tools/run-tests.py

```diff
@@ -128,6 +128,8 @@ def get_binary_path(bin_dir_path):
             ['--jerry-libc=off', '--compile-flag=-m32', '--cpointer-32bit=on', '--system-allocator=on']),
     Options('buildoption_test-external_context',
             ['--jerry-libc=off', '--external-context=on']),
+    Options('buildoption_test-snapshot_tool',
+            ['--jerry-cmdline-snapshot on']),
```

Comment:
be consistent with existing code, use `--jerry-cmdline-snapshot=on`. (might even be erroneous to use a space here, as the list should contain command line arguments already separated.)